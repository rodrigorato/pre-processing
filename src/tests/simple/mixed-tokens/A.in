public class Foo {
    public void foo(String[] args) {
    	var s = new String(#"#{args[0]}");
    }
}