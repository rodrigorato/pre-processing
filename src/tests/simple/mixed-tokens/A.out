public class Foo {
    public void foo(String[] args) {
    	String s = new String("" + (args[0]) + "");
    }
}